crates/analyzer/src/handlers/create_reference.rs

```diff
@@ -166,6 +325,132 @@ impl Handler for CreateReference<'_> {
 }
 
 impl VerylGrammarTrait for CreateReference<'_> {
+    fn veryl(&mut self, arg: &Veryl) -> Result<(), ParolError> {
```

Comment:
How about following the function order of `VerylGrammarTrait` implementation.

Reply:
I changed function order.